### PR TITLE
[FIX] pos_order_mgmt: loaded orders date to locale

### PR DESCRIPTION
**Pull request by chienandalu**
- Loaded orders where in UTC

cc @Tecnativa

### CODE REVIEW

File: pos_order_mgmt/static/src/js/widgets.js [hunk 1]
```diff
@@ -316,6 +316,14 @@ odoo.define('pos_order_mgmt.widgets', function (require) {
                 args: [query || '', this.pos.pos_session.id],
             }).then(function (result) {
                 self.orders = result;
+                // Get the date in local time
+                _.each(self.orders, function (order) {
+                    if (order.date_order) {
+                        var date = moment.utc(order.date_order).toDate();
+                        order.date_order = moment(date).format(
+                            'YYYY-MM-DD HH:mm:ss');
```
Comment (Tardo) [inline pos_order_mgmt/static/src/js/widgets.js]: Here can use ```moment.utc(order.date_order).local().format('YYYY-MM-DD HH:mm:ss')```

**Review by Tardo — APPROVED** [review]
LGTM, code review :+1:

File: pos_order_mgmt/static/src/js/widgets.js (hunk 1 above)
Comment (chienandalu) [reply pos_order_mgmt/static/src/js/widgets.js]: Thanks @Tardo I put it as you said :)

**Comment by pedrobaeza** [comment]
I would say Odoo has already functions for doing this directly (or why don't you pass tz on RPC for getting the date from server directly in proper timezone?)

**Comment by chienandalu** [comment]
I'd pass a context if I had any, but I don't

**Review by pedrobaeza — APPROVED** [review]
OK then

**Pull request merged**